src/main/java/cope/cosmos/client/features/modules/visual/NewChunks.java

```diff
@@ -3,28 +3,39 @@
 import cope.cosmos.client.events.PacketEvent;
 import cope.cosmos.client.features.modules.Category;
 import cope.cosmos.client.features.modules.Module;
+import cope.cosmos.client.features.modules.client.Colors;
 import cope.cosmos.client.features.setting.Setting;
 import cope.cosmos.event.annotation.Subscription;
 import cope.cosmos.util.render.RenderBuilder;
 import cope.cosmos.util.render.RenderUtil;
 import io.netty.util.internal.ConcurrentSet;
+import net.minecraft.client.renderer.culling.Frustum;
 import net.minecraft.network.play.server.SPacketChunkData;
-import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.AxisAlignedBB;
+import net.minecraft.util.math.Vec2f;
 
 import java.awt.*;
 import java.util.Set;
 
+import static cope.cosmos.util.render.RenderBuilder.Box;
+
 public class NewChunks extends Module {
+    public static NewChunks INSTANCE;
+
     public NewChunks() {
-        super("NewChunks", Category.VISUAL, "Highlights newly generated chunks", () -> String.valueOf(newChunks.size()));
+        super("NewChunks", Category.VISUAL, "Highlights newly generated chunks");
+        INSTANCE = this;
     }
 
-    public static final Setting<LimitedBox> box = new Setting<>("Box", LimitedBox.OUTLINE).setDescription("How to draw the box");
+    public static final Setting<LimitedBox> box = new Setting<>("Render", LimitedBox.OUTLINE).setDescription("How to draw the box");
+    public static final Setting<Double> height = new Setting<>("Height", 0.0, 0.0, 3.0, 1).setDescription("The height to render the new chunk at");
 
-    public static final Setting<Double> outlineWidth = new Setting<>("Width", 0.0, 1.5, 3.0, 1).setDescription("Line width of the outline render").setVisible(() -> box.getValue().equals(LimitedBox.FILLED) || box.getValue().equals(LimitedBox.BOTH) || box.getValue().equals(LimitedBox.OUTLINE) || box.getValue().equals(LimitedBox.CLAW));
+    public static final Setting<Double> outlineWidth = new Setting<>("Width", 0.0, 1.5, 3.0, 1).setDescription("Line width of the outline render").setVisible(() -> box.getValue().equals(LimitedBox.BOTH) || box.getValue().equals(LimitedBox.OUTLINE) || box.getValue().equals(LimitedBox.CLAW));
     public static final Setting<Color> color = new Setting<>("Color", new Color(255, 0, 0, 45)).setDescription("The highlight color");
```

Comment:
Remove both the Sync and Color settings, I don't want individual modules to have color settings, unless it's necessary for the functionality